### ISSUE TITLE
Whether the project was abandoned

### ISSUE THREAD

**Issue by selfshore** [alekseyl1992/async_http2#1]

**Comment by alekseyl1992** [comment]
I was planning to use it in one of my projects, but there was some timeout-related problems. I haven't looked at it yet. So I'm not sure whether it was caused by the library or by the external server issues.
But I didn't really plan to interface all the h2 features into Python anyways.

Btw. why are you asking?

**Comment by selfshore** [comment]
> I was planning to use it in one of my projects, but there was some timeout-related problems. I haven't looked at it yet. So I'm not sure whether it was caused by the library or by the external server issues. But I didn't really plan to interface all the h2 features into Python anyways.
> 
> Btw. why are you asking?

I want to use it to crawl data,I think it's better than HTTPX in some ways and look forward to its improvement

**Comment by alekseyl1992** [comment]
Yes, the reason I've created this library is that httpx has (or at least had) issues with parallel requests handling: multiple streams inside one HTTP/2 connection were locking each other's results.
I've created a discussion in their repo, but got no answers: https://github.com/encode/httpx/discussions/1766
